crates/message/src/message.rs

```diff
@@ -827,6 +827,12 @@ impl UserData for Message {
                     .map_err(any_err)?)
             },
         );
+        methods.add_method(
+            "remove_all_named_header",
```

Comment:
Let's pluralize this; I can see that the underlying rust method isn't pluralized and that seems like a typo that I made.  If you feel like pluralizing the rust code while you're looking at this, that would be appreciated!

```suggestion
            "remove_all_named_headers",
```